test/fixtures/v1-vt-05401-state-utility-lowincome.json

```diff
@@ -444,35 +572,60 @@
       "owner_status": [
         "homeowner"
       ],
-      "start_date": "2023",
+      "start_date": "2023-08-01",
       "ami_qualification": null,
       "eligible": true,
       "short_description": "Work with an Efficiency Excellence Network contractor to improve your home's insulation and air sealing and get 75% off project costs up to $9,500."
     },
+    {
+      "payment_methods": [
+        "rebate"
+      ],
+      "authority_type": "utility",
+      "authority": "vt-burlington-electric-department",
+      "program": "Burlington Electric Department - Lawn Care - Residential Rebate",
+      "program_url": "https://www.burlingtonelectric.com/lawnmowers",
+      "item": {
+        "type": "electric_outdoor_equipment",
+        "name": "Electric Outdoor Equipment"
+      },
+      "amount": {
+        "type": "percent",
+        "number": 0.5,
+        "maximum": 300
+      },
+      "owner_status": [
+        "homeowner"
+      ],
+      "start_date": "2022-01-01",
+      "end_date": "2023-12-31",
+      "ami_qualification": null,
+      "eligible": true,
+      "short_description": "Up to $100 back for residential push mower or up to $300 back for a ride-on mower and $50 on electric chainsaws and trimmers."
+    },
     {
       "payment_methods": [
         "rebate"
       ],
       "authority_type": "state",
-      "authority": "vt-ev",
-      "program": "Efficiency Vermont Ground Source Heat Pump Rebate Program",
-      "program_url": "https://www.efficiencyvermont.com/rebates/list/ground-source-heat-pumps",
+      "authority": "vt-efficiency-vermont",
+      "program": "Efficiency Vermont Ground Source Heat Pump Rebate",
+      "program_url": "https://www.efficiencyvermont.com/Media/Default/docs/rebates/forms/efficiency-vermont-ground-source-heat-pump-rebate-form.pdf",
       "item": {
         "type": "geothermal_heating_installation",
         "name": "Geothermal Heating Installation"
       },
       "amount": {
         "type": "dollars_per_unit",
         "number": 2100,
-        "maximum": 50000,
         "unit": "ton"
       },
       "owner_status": [
         "homeowner"
       ],
-      "start_date": "2021",
+      "start_date": "2021-07-01",
       "ami_qualification": null,
-      "eligible": true,
+      "eligible": false,
```

Comment:
This is now false because it's superseded by BED's own GSHP incentive on line 659.